recipe/meta.yaml

```
@@ -28,12 +27,15 @@ outputs:
       run:
         - python
         - googleapis-common-protos >=1.56.2,<2.0dev
```

```suggestion
        - googleapis-common-protos >=1.56.2,<2.0dev0
```
https://github.com/googleapis/python-api-core/blob/v2.19.1/setup.py#L32

---

addressed, thanks